CMakeLists.txt

```diff
@@ -31,15 +31,24 @@
 # along with Vega Strike. If not, see <https://www.gnu.org/licenses/>.
 
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.16 FATAL_ERROR)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.21 FATAL_ERROR)
+
+IF (POLICY CMP0087)
+    cmake_policy(SET CMP0087 NEW)
+ENDIF ()
+SET(X_VCPKG_APPLOCAL_DEPS_INSTALL ON)
+
+IF (POLICY CMP0167)
+    CMAKE_POLICY (SET CMP0167 OLD)
+ENDIF (POLICY CMP0167)
 
 # There are a number of custom CMake packages
 # Tell CMake where to find them
 SET(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
 IF (WIN32)
     SET(VCPKG_MANIFEST_MODE ON)
-    SET(VCPKG_MANIFEST_DIR "${CMAKE_SOURCE_DIR}/cmake")
+    SET(VCPKG_MANIFEST_DIR "${CMAKE_SOURCE_DIR}")
```

Comment:
this is fine; as part of my testing to figure out what was wrong with the Windows CI build I moved them in with the CMake files; would be nice to do something similar to help clean up the root, but ultimately just looking to get it working right now.